client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-source-url/index.tsx

```diff
@@ -7,23 +7,28 @@ import { useAnalyzeUrlQuery } from 'calypso/data/site-profiler/use-analyze-url-q
 import { useSiteSlug } from 'calypso/landing/stepper/hooks/use-site-slug';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import wpcom from 'calypso/lib/wp';
-import type { Step } from '../../types';
+import type { StepProps } from '../../types';
 import type { UrlData } from 'calypso/blocks/import/types';
 
 import './style.scss';
-interface Props {
+
+interface SourceSiteInputProps {
 	hasError?: boolean;
 	onComplete: ( siteInfo: UrlData ) => void;
+	title?: string;
+	subTitle?: string;
 }
 
-export const SourceSiteInput: FC< Props > = ( { onComplete } ) => {
+export const SourceSiteInput: FC< SourceSiteInputProps > = ( { onComplete, ...options } ) => {
```

Comment:
WDYT to use `...props` instead of `...options`? It seems more standardized and clear that it's the component props.

Reply:
I usually prefer options when the object represents just part of a subset of props, but I have no strong opinion about it.
I updated here [f7fbb85](https://github.com/Automattic/wp-calypso/pull/93487/commits/f7fbb85b9fa7990d0237f0ac29edeb6ec7cc02d5)